lib/ansible/plugins/connection/httpapi.py

```diff
@@ -190,10 +190,9 @@ def __init__(self, play_context, new_stdin, *args, **kwargs):
         self._auth = None
 
         if self._network_os:
-
             self.httpapi = httpapi_loader.get(self._network_os, self)
             if self.httpapi:
-                self._sub_plugin = {'type': 'httpapi', 'name': self._network_os, 'obj': self.httpapi}
+                self._sub_plugin.extend([{'type': 'httpapi', 'name': self._network_os, 'obj': self.httpapi}])
```

Comment:
Any particular reason why using `extend` instead of `append`?
```suggestion
                self._sub_plugin.append({'type': 'httpapi', 'name': self._network_os, 'obj': self.httpapi})
```

Reply:
Fixed